packages/drivers/odsp-driver/src/createFile.ts

```diff
@@ -164,13 +164,19 @@ function extractShareLinkData(
 	return shareLinkInfo;
 }
 
+function ensureStartsWithSlash(path: string): string {
```

Comment:
make it path: string | undefined and do the check or undefined here once, instead of each time this function is called.